Contributing.md

```diff
@@ -0,0 +1,69 @@
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+# Pull Request Process
+1.	Fork the repository and create a separate branch regarding your contribution.
+2.	Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations, and container parameters.
+3.	Issue a pull request and ask us to review it. Don't forget to link the PR to a relevant issue if it exists.
+4.	Make sure the pull request passes all checks
+5.	You may merge the Pull Request once you have the sign-off of two other developers, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
+
```

Comment:
I'm missing a possible section about how should a good code-review look.
also, please note (and add it to the guide) that PRs that do not pass the future CI\tests\checks or have conflicts - may not get reviewed until those are solved.